databricks/koalas/missing/common.py

```diff
@@ -16,32 +16,36 @@
 
 
 memory_usage = lambda f: f(
-    'memory_usage',
+    "memory_usage",
     reason="Unlike pandas, most DataFrames are not materialized in memory in Spark "
-           "(and Koalas), and as a result memory_usage() does not do what you intend it "
-           "to do. Use Spark's web UI to monitor disk and memory usage of your application.")
+    "(and Koalas), and as a result memory_usage() does not do what you intend it "
```

Comment:
and, this actually looks a bit odd. Because alined indentation within parentheses is legitimate per PEP 8 as well (i use this style quite often too)